client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -307,4 +305,45 @@ public void testEndpoint() {
         assertEquals("/a/b/_create", Request.endpoint("a", "b", "_create"));
         assertEquals("/a/b/c/_create", Request.endpoint("a", "b", "c", "_create"));
     }
-}
\ No newline at end of file
+
+    private void enrichReplicationRequest(ReplicatedWriteRequest request, Map<String, String> expectedParams) {
```

Comment:
I'd have different static methods `setRandomTimeout(..)` and `setRandomRefreshPolicy(...)`